change cinder backend lvm to backend ceph

wchy1001 opened the pull request:
Issue: 39240
http://172.16.30.125/issues/39240
1. remove unused disks
2. add ceph osd disks
3. label osd disks
3. deal with ceph in kolla.

Commit:
change cinder backend lvm to backend ceph

Issue: 39240
1. remove unused disks
2. add ceph osd disks
3. label osd disks
3. deal with ceph in kolla.

wu-wenxiang commented on src/ansible-build-openstack-env/playbooks/roles/init03-cobbler/tasks/main.yml:
```diff
@@ -127,17 +127,20 @@
+           --disk {{ vm_system_path }}/{{ hostvars[item]['hostname'] }}.qcow2,format=qcow2,bus=scsi
+           --disk size=30,bus=scsi
+           --disk size=30,bus=scsi
+           --disk size=30,bus=scsi
```
这里 size = 30，不要hardcode，放到 group_vars/all，或者 inventory

Commit:
support both lvm and ceph

1. chose cinder backend in group/all.yml file

http://172.16.30.125/issues/39240
